model/model.go

```diff
@@ -41,9 +46,13 @@ const (
 
 	// FeeTypeAmount represent FeeType in amount
 	FeeTypeAmount FeeType = "amount"
+
+	// RequestIDContextKey contact that holds the RequestID context key for app to storage later pass through
+	RequestIDContextKey Key = "penta_RequestIDContextKey"
```

Comment:
not sure about this sir. why add `penta_`?

Reply:
This was to filter through requests to identify requests from consumer app.
On hindsight, we do not only have consumer app using our sdk.

Will update.